cli/src/config.rs

```diff
@@ -283,11 +281,30 @@ pub struct ConfigEnv {
 impl ConfigEnv {
     /// Initializes configuration loader based on environment variables.
     pub fn from_environment() -> Self {
+        let config_dir = if cfg!(target_os = "macos") {
+            etcetera::base_strategy::choose_native_strategy()
+                .ok()
+                .map(|s| {
+                    use etcetera::BaseStrategy as _;
+                    // note that etcetera calls Library/Application Support the "data dir",
+                    // Library/Preferences is supposed to be exclusively plists
+                    s.data_dir()
+                })
+        } else {
+            etcetera::choose_base_strategy().ok().map(|s| {
+                use etcetera::BaseStrategy as _;
```

Comment:
nit: just import globally unless we have conflict resolution.

Reply:
do I have to? I hate that for traits...